src/main/java/com/google/codeu/data/Message.java

```diff
@@ -52,36 +53,28 @@ public Message(UUID id, String user, String text, long timestamp, String recipie
     this.recipient = recipient;
   }
 
-  /**
-   * @return current non-null UUID
-   */
+  /** @return current non-null UUID */
```

Comment:
Add a summary for this.  "Returns a current non-null UUID." is a valid fix.

Reply:
Do you mean change it to /**Returns a current non-null UUID*/ or 
/**Returns a current non-null UUID.
* @retunrs current non-null UUID*/

Reply:
You unfortunately need to double the text here to make the linter happy.

So yes, you'e right.